src/components/CreditCardBilling/colorPallete.ts

```diff
@@ -0,0 +1,12 @@
+import { amber, indigo, tomato } from '@radix-ui/colors';
+import type { Category } from './CreditCardBilling';
+
+type CreditCardBillingPallete = {
+  [key in Category]: React.CSSProperties['color'];
+};
+
+export const pallete: CreditCardBillingPallete = {
+  shopping: tomato.tomato9,
+  transfer: indigo.indigo9,
+  food: amber.amber9,
+};
```

Comment:
```suggestion
export const pallete: CreditCardBillingPallete = {
  shopping: undefined,
  transfer: indigo.indigo9,
  food: amber.amber9,
};
```
현재의 선언대로라면 컬러 값에 undefined도 포함될 수 있는데, 혹시 의도하신게 맞을까요??